scripts/test-wrapper.js

```
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
+// Script to run tests after commit and generate coverage reports
+const { execSync, spawnSync } = require("child_process");
+const { execSync } = require("child_process");
```

Biome (1.9.4)

[error] Shouldn't redeclare 'execSync'. Consider to delete it or rename it.

'execSync' is defined here:


(lint/suspicious/noRedeclare)